.github/workflows/publish.yml

```diff
@@ -2,7 +2,7 @@ name: Publish
 
 on:
   workflow_run:
-    workflows: ["ReleaseStart"]
+    workflows: ["SyncMain"]
```

Comment:
I think we should also change that in `App Distribute CI` workflow

Reply:
🤔  But don't we want to distribute our sample app on every commit in develop?